src/rtk/features/singleRights/singleRightsSlice.ts

```diff
@@ -34,31 +34,30 @@ interface detailParams {
   name: string;
   value: string;
 }
-
-interface chosenService {
+export interface ChosenService {
   accessCheckResponses?: delegationAccessCheckResponse[];
   service?: ServiceResource;
-  status?: 'Delegable' | 'NotDelegable';
+  status?: 'Delegable' | 'NotDelegable' | 'PartiallyDelegable';
   errorCode?: string;
 }
 
-interface chosenServices {
-  chosenServices: chosenService[];
+interface chosenServiceList {
```

Comment:
Is this the interface for the sliceState? If so, why is it called ChosenServiceList?

Reply:
ChosenService and ChosenServices is such a small difference and easy to mistake. Think ChooseServiceList is better to distinguish the difference than ChosenServices.

Reply:
Yeah, but when I look at the definition of the initial state, it looks like this interface here is actually the interface for the state itself, not just the chosenServices.

My comment here isn't really concerned with the name-change from chosenServices to chosenServiceList, I'm thinking that the interface should instead be called sliceState or something, because if I understand it correctly, that is what it is.
When we add new things to the state, such as f.ex. successfullDelegations or something, the fact that the state type is named chosenServicesList will make no sense at all.

(Sorry, this really is something I should have commented on when the slice was added more than here in this PR, but I didn't spot it until now)

Reply:
Oh yeah, should be named to SliceState